strawberryfields/ops.py

```diff
@@ -2032,6 +2026,43 @@ def _decompose(self, reg, **kwargs):
 shorthands = ['New', 'Del', 'Vac', 'Measure', 'MeasureX', 'MeasureP', 'MeasureHD', 'Fourier', 'RR',
               'All']
 
+
+class Wrapper:
+    """Wrapper class to modify the module level
+    attribute lookup.
+
+    This allows module attributes to be deprecated.
+
+    Current list of deprecated attributes:
+
+    * ``Measure``: instead use ``MeasureFock``
+
+    .. note::
+
+        With Python 3.7+, there is new support for a module-level
+        ``__getattr__`` function, which should enable this functionality
+        without needing to modify ``sys.modules``.
+    """
+    deprecation_map = {"Measure": "MeasureFock"}
+
+    def __init__(self, mod):
+        self.mod = mod
+
+    def __getattr__(self, name):
+        if name in self.deprecation_map:
```

Comment:
Is it conceivable that a shorthand (or other name) would be deprecated without a replacement? If so, perhaps the deprecation message belongs in `deprecation_map` as the value, that way it can be a little more versatile.

Reply:
Good point. I can't forsee that happening tbh, more so that we rename existing operations. @co9olguy?

Reply:
Seems unlikely that a shorthand would ever be deprecated without a replacement. I also don't foresee removing existing ops (though I guess it could happen). As @josh146 says, most likely thing is just a renaming (which is a form of deprecation I suppose; so far, we've just been doing it with breaking changes, but deprecating things when we change name is probably better option going forward).

Reply:
Okay, let's keep it simple for now, and if we ever have a more complicated deprecation we can change this class.